integrations/server/test_covidcast.py

```diff
@@ -10,12 +10,13 @@
 
 # first party
 from delphi_utils import Nans
+from delphi.epidata.acquisition.covidcast.covidcast_row import CovidcastRow
 from delphi.epidata.acquisition.covidcast.test_utils import CovidcastBase
 
 # use the local instance of the Epidata API
+# TODO: should we still be using this?
 BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
-
-
+IGNORE_FIELDS = ["id", "direction_updated_timestamp", "value_updated_timestamp", "source", "time_type", "geo_type"]
```

Comment:
I have beef with stuffing test materials in CovidcastRow (see below) but if we're going to do it, seems like this array belongs there too, and not as a file constant here